README.md

```diff
@@ -49,28 +49,28 @@ yarn add pulsar-client
 Then you can run the following simple end-to-end example:
 
 ```javascript
-const Pulsar = require('pulsar-client');
+const Pulsar = require("pulsar-client");
```

Comment:
The quotes were auto-corrected, maybe by a checker plugin. Does it make sense or shall I revert?

Reply:
It better uses `'`.

Reply:
You should revert it.

Reply:
Thanks. Resolved.